src/NUnitCommon/nunit.common/ClientTestResult.cs

```diff
@@ -6,13 +6,13 @@
 using System.Xml;
 using NUnit.TextDisplay;
 
-namespace NUnit.ConsoleRunner
+namespace NUnit
```

Comment:
Prefer `NUnit.Common`

Reply:
Let me know what you think about my rationale above.